### PR TITLE
Update README.md ARCH linux.

### PR DESCRIPTION
**Pull request by kylerschin**
Updated `pacman -S cjdns` to `pacman -Syu cjdns` and also added more info on the config file and starting/stopping the service, as well as added AUR info.

### CODE REVIEW

**Comment by kylerschin** [comment]
@cjdelisle is this pull request acceptable?

**Comment by kylerschin** [comment]
I just put the package name for AUR (Arch Linux User Repository), because I don't know what people use to install packages. I used yaourt and archaur but you guys will probably be: "You Son of a gun, I don't give a *******". LOL.

**Comment by fbt** [comment]
Why do you think that upgrading your whole system to install a package is a good default way of installing cjdns? If your sync db has the version of cjdns that is available from mirrors, it will get installed with no problems. If that version is so old it's not on the mirrors anymore, you'll get a 404 and will upgrade the system yourself.

**Comment by kylerschin** [comment]
@fbt I fixed it. Can I have it merged???

**Comment by kylerschin** [comment]
I'm stupid lol.

File: README.md [hunk 1]
```diff
@@ -120,11 +120,11 @@ Everything you need is available prebuild in FreeBSD' ports.
 
 You can install cjdns by running
 
-    pacman -S cjdns
+    pacman -Sy cjdns
```
Comment (fbt) [inline README.md]: What the fuck.
-Sy is dangerous! Never tell people to use it!
http://gist.io/5660494

**Pull request merged**